plexapi/photo.py

```diff
@@ -158,6 +172,7 @@ def sync(self, resolution, client=None, clientId=None, limit=None, title=None):
         sync_item.contentType = self.listType
         sync_item.metadataType = self.METADATA_TYPE
         sync_item.machineIdentifier = self._server.machineIdentifier
+        sync_item.userRating = self.userRating
```

Comment:
Let me repeat my question: why are you assigning `userRating` to `sync_item` here? The `SyncItem` class knows nothing about this field and therefore it wouldn't be processed anyway. Have you misplaced the assignment and there is a place really waiting for it?

Maybe you could be polite yourself and answer the simple and not rude in any way question without throwing a tantrum? You're in the Internet, man, there are a lot of different cultures (in some of them being straightforward is the proper way of doing things, not the rude one) and not everybody here would like to offend you. Not marking conversation as resolved when it isn't seems for me like a good idea as well.